test/JDBC/expected/babel_context_info-vu-prepare.out

```diff
@@ -0,0 +1,14 @@
+CREATE PROCEDURE SET_CONTEXT_INFO
+    @context_info varbinary(128)
+AS
+BEGIN
+    EXEC sys.bbf_set_context_info @context_info
+END;
+GO
+
+CREATE PROCEDURE GET_CONTEXT_INFO
```

Comment:
why do we need this procedure?

Reply:
GET_CONTEXT_INFO and SET_CONTEXT_INFO is for upgrade test.